restore/README.md

```diff
@@ -0,0 +1,130 @@
+# Restore action
+
+The restore action restores cache just like the main `cache` action, except it doesn't save the cache. So for cases where users only want to restore cache and not save anything in return, this action is pretty helpful. Moreover, this action accepts the same set of inputs as the `cache` action.
+
+## Inputs
+
+* `path` - A list of files, directories, and wildcard patterns to cache and restore. See [`@actions/glob`](https://github.com/actions/toolkit/tree/main/packages/glob) for supported patterns.
+* `key` - String used while saving cache for restoring the cache
+* `restore-keys` - An ordered list of prefix-matched keys to use for restoring stale cache if no cache hit occurred for key.
+> **Note**
+It is very important to use the same `key` and `path` that were used by either `actions/cache` or `actions/cache/save` while saving the cache.
+
+### Environment Variables
+* `SEGMENT_DOWNLOAD_TIMEOUT_MINS` - Segment download timeout (in minutes, default `60`) to abort download of the segment if not completed in the defined number of minutes. [Read more](https://github.com/actions/cache/blob/main/workarounds.md#cache-segment-restore-timeout)
+
+## Outputs
```

Comment:
May be outputs section should come before environment variables one.

Reply:
We are following the same sequence in the cache action readme as well.

Reply:
I was thinking from user perspective. Output is more common need than env. In a doc we should start with more commonly useful info at top and then go down to less common useful ones.

Reply:
I was thinking from user perspective. Output is more common need than env. In a doc we should start with more commonly useful info at top and then go down to less common useful ones.

Reply:
Updated